src/main/thrift/parquet.thrift

```diff
@@ -237,6 +237,110 @@ struct SizeStatistics {
    3: optional list<i64> definition_level_histogram;
 }
 
+/**
+ * Physical type and encoding for the geometry type.
+ */
+enum GeometryEncoding {
+  /**
+   * Allowed for physical type: BYTE_ARRAY.
+   *
+   * Well-known binary (WKB) representations of geometries.
+   *
+   * To be clear, we follow the same rule of WKB and coordinate axis order from
+   * GeoParquet [1][2]. Geometries SHOULD be encoded as ISO WKB [3][4]
+   * supporting XY, XYZ, XYM, XYZM and the standard geometry types
+   * Point, LineString, Polygon, MultiPoint, MultiLineString, MultiPolygon,
+   * and GeometryCollection). Coordinate order is always (x, y) where x is
+   * easting or longitude and y is northing or latitude. This ordering explicitly
+   * overrides the axis order as specified in the CRS following the GeoPackage
+   * specification [5].
+   *
+   * This is the preferred encoding for maximum portability. It also supports
+   * GeometryStatistics to be set in the column chunk and page index.
+   *
+   * [1] https://github.com/opengeospatial/geoparquet/blob/v1.1.0/format-specs/geoparquet.md?plain=1#L92
+   * [2] https://github.com/opengeospatial/geoparquet/blob/v1.1.0/format-specs/geoparquet.md?plain=1#L155
+   * [3] https://portal.ogc.org/files/?artifact_id=18241
+   * [4] https://www.iso.org/standard/60343.html
+   * [5] https://www.geopackage.org/spec130/#gpb_spec
+   */
+  WKB = 0;
+}
+
+/**
+ * Interpretation for edges of elements of a GEOMETRY logical type. In other
+ * words, whether a point between two vertices should be interpolated in
+ * its XY dimensions as if it were a Cartesian line connecting the two
+ * vertices (planar) or the shortest spherical arc between the longitude
+ * and latitude represented by the two vertices (spherical). This value
+ * applies to all non-point geometry objects and is independent of the
+ * coordinate reference system.
+ *
+ * Because most systems currently assume planar edges and do not support
+ * spherical edges, planar should be used as the default value.
+ */
+enum Edges {
+  PLANAR = 0;
+  SPHERICAL = 1;
+}
+
+/**
+ * Bounding box of geometries in the representation of min/max value pair of
+ * coordinates from each axis when Edges is planar. Values of Z and M are omitted
+ * for 2D geometries. When Edges is spherical, the bounding box is in the form of
+ * [westmost, eastmost, southmost, northmost], with necessary min/max values for
+ * Z and M if needed.
```

Comment:
## On the "Edges" name

> > Rename EdgeInterpolation back to Edges since the explanation in comments already made it clear and we don't have to use the long name any more.

> +1 (I care that spherical edges can be represented and that it's clear what that means, but the name doesn't matter much to me)

I think that the "edges" name is quite problematic. Such simple name assumes that no other information about edges will never be added. What if a future version wants to add information about edge interpolation accuracy, or the time period of each edge (as in moving features), _etc_?

I think we should reserve "edges" for the future. If not for edges data, it may be for something similar to CSS, where `border` is a shortcut for other properties such as `border-style`, `border-width`, _etc._

Another reason is that developers rarely read documentation until they have no choice. For someone who see the data without having read the documentation, "edges: planar" is highly confusing. Even peoples who read the documentation may not remember all details.

`EdgeInterpolation`, `InterpolationMethod` or simply `Interpolation` would also be more consistent with other OGC standards such as OGC 16-140 — Moving Features, which already uses "interpolation". See in particular "_Predefined Interpolation Methods"_ below [section 6.3.1](https://docs.ogc.org/bp/16-140r1/16-140r1.html#simple). 

## On the "planar" and "spherical" names
As said before, I think that `planar` and `spherical` doesn't mean anything in the context of interpolations. For the "planar" case, see instead above-cited Moving features specification: they already provide `discrete`, `stepwise`, `linear` and `spline` interpolation methods. The two first ones may not be applicable to 2D space, but `linear` and `spline` are applicable.

Reply:
(and also GEOS (and PostGIS/shapely/sf/... based on it) uses the "interpolate" term constructing a point on a line segment)